velox/common/memory/SharedArbitrator.cpp

```diff
@@ -219,10 +253,25 @@ uint64_t SharedArbitrator::shrinkCapacity(
   std::vector<Candidate> candidates = getCandidateStats(pools);
```

Comment:
Add guard to reclaim the freedBytes in case of exception thrown in this function

Reply:
What exceptions can be thrown in reclaim free?